src/Symfony/Bundle/WebServerBundle/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/web-server-bundle",
+    "type": "web-server-bundle",
+    "description": "Symfony WebServerBundle",
+    "keywords": [],
```

Comment:
It would be good to add some keywords for discoverability through the Packagist search